defaults/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+# defaults file for sanity-check
+
+SANITY_CHECK_DISK_SPACE:
+    # Mount point
+  - mount: /
+    # Treshold in mb
+    treshold_mb: 5000
```

Comment:
Maybe worth using % instead of an actual number of MB, as this is going to be very dependent on the actual size of the drive. Maybe a default threshold of 80% of the hard drive space?